### PR TITLE
Add arc deployment contributions to arc extension

### PR DESCRIPTION
**Pull request by Charles-Gagnon**

### CODE REVIEW

File: extensions/arc/notebooks/arcDeployment/deploy.arc.control.plane.ipynb [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/src/sql/media/microsoft-small-logo.png)\n",
```
Comment (alanrenmsft) [inline extensions/arc/notebooks/arcDeployment/deploy.arc.control.plane.ipynb:26]: are you planning to update the icons to the use the one included in this extension?

File: extensions/arc/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "icon": "images/extension.png",
   "engines": {
     "vscode": "*",
-    "azdata": ">=1.19.0"
+    "azdata": ">=1.20.0"
```
Comment (alanrenmsft) [inline extensions/arc/package.json:12]: are we relying on any new changes in ADS?

Comment (Charles-Gagnon) [reply extensions/arc/package.json:12]: Not for the deployment - but Brian made some fixes to the model view stuff so this extension will need the latest ADS version.

File: extensions/arc/notebooks/arcDeployment/deploy.arc.control.plane.ipynb [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/arc/images/microsoft-small-logo.png)\n",
+                " \n",
+                "## Deploy Azure Arc Data Controller on an existing cluster\n",
+                " \n",
+                "This notebook walks through the process of deploying a <a href=\"https://azure.microsoft.com/services/azure-arc/\">Azure Arc controller</a> on an existing cluster.\n",
+                " \n",
+                "* Follow the instructions in the **Prerequisites** cell to install the tools if not already installed.\n",
+                "* The **Required information** will check and prompt you for password if it is not set in the environment variable. The password can be used to access the data controller.\n",
+                "\n",
+                "<span style=\"color:red\"><font size=\"3\">Please press the \"Run Cells\" button to run the notebook</font></span>"
```
Comment (alanrenmsft) [inline extensions/arc/notebooks/arcDeployment/deploy.arc.control.plane.ipynb:35]: Just realized you have to update the text to "Run all" to reflect the recent ADS changes

**Pull request merged**